cpp/src/gandiva/greedy_dual_size_cache.h

```diff
@@ -110,6 +114,25 @@ class GreedyDualSizeCache {
     }
   }
 
+  void InsertObjectCode(const Key& key, const ValueCacheObject<Value>& value) {
```

Comment:
why are you adding these new methods?

Reply:
These methods was added just to preserve the old ones so I can benchmak switching between them instead of switching the project branchs

Reply:
Can you remove them now since you already benchmarked?